bundles/org.openhab.binding.teleinfo/src/main/resources/OH-INF/i18n/teleinfo.properties

```diff
@@ -70,10 +283,22 @@ channel-type.teleinfo.couleurDemainType.description = Following day color
 channel-type.teleinfo.couleurDemainType.state.option.Bleu = Bleu
 channel-type.teleinfo.couleurDemainType.state.option.Blanc = Blanc
 channel-type.teleinfo.couleurDemainType.state.option.Rouge = Rouge
+channel-type.teleinfo.currentType.label = Teleinfo Current Type
+channel-type.teleinfo.currentType.label = Teleinfo Current Type
+channel-type.teleinfo.currentType.label = Teleinfo Current Type
```

Comment:
It looks like you have several duplicated entries in the "channel types" section.
You have 7 channel types appearing 3 times.
Please check what would be really expected.
Would it be a bug in the i18n tool ?

Reply:
I've removed the duplicated lines. 
I think the i18n tool generates these entries because identical channel types are defined in several xml files.

Reply:
You should not have a channel type defined several times. That is certainly the root cause and what should be fixed. The common solution is to put channel types in a film named channels.xml.

Reply:
Yes I probably should do some clean up in the xml config files.

I wanted to have a separate xml file for each `channel-group-type` and it seems that `channel-group-type` cannot use a `channel-type` from another file so I end up duplicating `channel-type` in each xml file.

What you suggest is to aggregate every `channel-group-type` and `channel-type` in the same xml file? Isn't there a way to keep separate files?

Reply:
I am not aware of this limit. 
I have not played with groups of channels very often. 
I could be wrong but I think I already saw channels groups and channel types defined in separate files. Let me search an example.

Reply:
You can have a look at the NibeUplink Binding: `channel-group-type` and `channel-type` are defined in different XML files.
Apparently there is no constraint to have both in the same XML file.

Reply:
Thank you! The problem should have been located between chair and keyboard, as usual. I've submitted a new PR: #12204.